### ISSUE TITLE
windows binaries?

### ISSUE THREAD

**Issue by Spongman** [libimobiledevice/libimobiledevice#555]
where are the windows binaries?

**Comment by qmfrederik** [comment]
I don't think this project ships Windows binaries; however Quamotion (the company I work for) provides Windows binaries for libimobiledevice here: http://docs.quamotion.mobi/en/latest/imobiledevice/download.html. They are based on the code in github.com/libimobiledevice-win32.
